src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -24,7 +28,7 @@ public static class CLIOptions
             ArgumentName = "--log-console",
             ArgumentShortName = "-l <logLevel>",
             ArgumentDescription = "Sets the logging level | Options [error, warning, info (default), debug, trace]",
-            DefaultValue = "Info",
+            DefaultValue = "",
```

Comment:
The default value for log-console is not empty string

Reply:
Yes. It's "info". But in code empty string value is handled as default  --log-console value
https://github.com/stryker-mutator/stryker-net/blob/f74f782fee9aca1ce3beb1e3bce05dadc4daa711/src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs#L97-L106

Having empty value here will let easier to change default logging optoin (if this will ever be needed).

On other hand using value from default `StrykerOptions` requires additional implementation, because string representation of `LogEventLevel.Information` is different from "info".
This can be solved either:
 * renaming `LogEventLevel.Information` to `LogEventLevel.Info` either
 * adding additional string parsing in `StrykerOptions.ValidateLogLevel`: `"information"` or
 * adding some attribute with string representation (seems to complicated for this case to me)

What is your preference? Or maybe there is some more elegant way?

Reply:
LogEventLevel is a serilog concept, we can't change this. I would prefer the option to just be hard coded string info in this case instead of empty string because at least that says something about the option.